apps/native-component-list/screens/TextToSpeechScreen.js

```diff
@@ -107,6 +118,14 @@ export default class TextToSpeechScreen extends React.Component {
     );
   }
 
+  _getRandomVoice = () => {
+    if (Math.random() > 0.5) {
```

Comment:
Maybe we could take a random voice from the list of available voices? 🤔 Just thinking out loud 😄